hail/python/test/hail/linalg/test_linalg.py

```diff
@@ -745,7 +744,7 @@ def test_diagonal_sparse(self):
         bm = BlockMatrix.from_numpy(nd, block_size=2)
         bm = bm.sparsify_row_intervals([0, 0, 0, 0, 0], [2, 2, 2, 2, 2])
 
-        self.assertTrue(bm.is_sparse)
+        # self.assertTrue(bm.is_sparse)  # FIXME doesn't work in service
```

Comment:
Because is_sparse calls into the jvm: [source](https://github.com/chrisvittal/hail/blob/646de3c63dd2ecf05e0e630071f103123c1f3c3f/hail/python/hail/linalg/blockmatrix.py#L1245-L1258)

Reply:
Ah right. So to fix this we need `tblockmatrix` in python to track sparsity? It's okay not to fix in this PR, just want to be sure I understand what still needs to be done. Could you at least add an `is_sparse` test with `@fails_service_backend`, with a comment to uncomment these lines when it's fixed, to be sure we don't lose track of it?

Reply:
done